src/react-native/react-native-wizard.ts

```diff
@@ -0,0 +1,256 @@
+  addRNSentryGradlePlugin,
+  writeAppBuildGradle,
+} from './gradle';
+import { runReactNativeUninstall } from './uninstall';
+import { APP_BUILD_GRADLE, XCODE_PROJECT, getFirstMatchedPath } from './glob';
+import { ReactNativeWizardOptions } from './options';
+import { SentryProjectData } from '../utils/types';
+import {
+  addSentryInitWithSdkImport,
+  doesJsCodeIncludeSdkSentryImport,
+} from './javascript';
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+const xcode = require('xcode');
+
+export const RN_SDK_PACKAGE = '@sentry/react-native';
+
+export const RN_PACKAGE = 'react-native';
+export const RN_HUMAN_NAME = 'React Native';
+
+export const SUPPORTED_RN_RANGE = '>=0.69.0';
+
+export async function runReactNativeWizard(
+  options: ReactNativeWizardOptions,
+): Promise<void> {
+  options.telemetryEnabled = false;
+
+  if (options.uninstall) {
+    return runReactNativeUninstall(options);
+  }
+
+  printWelcome({
+    wizardName: 'Sentry React Native Wizard',
+    promoCode: options.promoCode,
+    telemetryEnabled: options.telemetryEnabled,
+  });
+
+  await confirmContinueIfNoOrDirtyGitRepo();
+
+  const packageJson = await getPackageDotJson();
+
+  await ensurePackageIsInstalled(packageJson, RN_PACKAGE, RN_HUMAN_NAME);
+
+  const rnVersion = getPackageVersion(RN_PACKAGE, packageJson);
+  if (rnVersion) {
+    await confirmContinueIfPackageVersionNotSupported({
+      packageName: RN_HUMAN_NAME,
+      packageVersion: rnVersion,
+      packageId: RN_PACKAGE,
+      acceptableVersions: SUPPORTED_RN_RANGE,
+    });
+  }
+
+  const { selectedProject, authToken } = await getOrAskForProjectData(
+    options,
+    'react-native',
+  );
+
+  await installPackage({
+    packageName: RN_SDK_PACKAGE,
+    alreadyInstalled: hasPackageInstalled(RN_SDK_PACKAGE, packageJson),
+  });
+
+  addSentryInit({ dsn: selectedProject.keys[0].dsn.public });
+
+  if (fs.existsSync('ios')) {
+    await patchXcodeFiles({ authToken });
+  }
+
+  if (fs.existsSync('android')) {
+    await patchAndroidFiles({ authToken });
+  }
+
+  const confirmedFirstException = await confirmFirstSentryException(
+    selectedProject,
+  );
+
+  if (confirmedFirstException) {
+    clack.outro(
+      `${chalk.green('Everything is set up!')}
+
+   ${chalk.dim(
+     'If you encounter any issues, let us know here: https://github.com/getsentry/sentry-react-native/issues',
+   )}`,
+    );
+  } else {
+    clack.outro(
+      `${chalk.dim(
+        'Let us know here: https://github.com/getsentry/sentry-react-native/issues',
+      )}`,
+    );
+  }
+}
+
+function addSentryInit({ dsn }: { dsn: string }) {
+  const prefixGlob = '{.,./src}';
+  const suffixGlob = '@(j|t|cj|mj)s?(x)';
+  const universalGlob = `App.${suffixGlob}`;
+  const jsFileGlob = `${prefixGlob}/+(${universalGlob})`;
+  const jsPath = getFirstMatchedPath(jsFileGlob);
+  if (!jsPath) {
+    clack.log.warn(
+      `Could not find main App file using ${chalk.bold(jsFileGlob)}.`,
+    );
```

Comment:
l: If this happens, do users need to do something manually? If yes, can we tell them what to do (add a link to docs or provide copy/paste code)?

Reply:
I'll add copy paste code.